src/lib/statistics/statistics_objects/null_value_ratio_statistics.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include "abstract_statistics_object.hpp"
+
+namespace opossum {
+
+// A single float value as an AbstractStatisticsObject, to keep architectures consistent....
+class NullValueRatioStatistics : public AbstractStatisticsObject {
```

Comment:
I believe we changed "null value ratio" to "null ratio" somewhere above. I find it more concise, how about you?

Reply:
"null ratio" was used in only three places and I changed it to "null value ratio" there. We have `ValueSegment::_values` and `ValueSegment::_null_values` and use the same naming scheme elsewhere, too. So I think its appropriate to talk about a "null value ratio"